sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/csv/CSVSuite.scala

```diff
@@ -396,6 +402,46 @@ class CSVSuite extends QueryTest with SharedSQLContext with SQLTestUtils {
     }
   }
 
+  test("SPARK-13543 Set explicitly the output as uncompressed") {
```

Comment:
Yes, which one would be better?
Should I just write like `write the uncompressed output via option`?